docs/getting-started/key-concepts/looping.md

```diff
@@ -81,6 +81,7 @@ There are a limited number of nodes and operations where you need to design a lo
 	* **List**: This operation executes only once, not for each incoming item.
 * [**Google Sheets**](../../nodes/nodes-library/nodes/GoogleSheets/README.md) node:
 	* **Read**: This operation will execute only once for the `Sheet` resource.
```

Comment:
```suggestion
	* **Read**: This operation executes only once for the `Sheet` resource.
```

Reply:
Looks like this change request isn't related to this PR. Maybe best to handle this separately?

Reply:
I'm hoping to sneak in style changes as we do other work.